registrationConfig.html

```diff
@@ -41,9 +48,40 @@ <h2>Map</h2>
             <div id="exportedAddins">
                 <h2>Addins</h2>
                 <div class="description">You have <span class="bold">not configured any addins</span>.</div>
-                <label id="includeThisAddin" class="hidden">
-                    <input type="checkbox" value="includeThisAddin"></input>
-                    <span class="description">Do not include this add-in into config</span>
+                <label id="exportAllAddins">
+                    <input id="export_all_addins_checkbox" value="export_all_addins" type="checkbox" class="checkboxStyle" checked></input>
+                    <span class="infoText">Export all addins</span>
+                    <p><b>Note:</b> Uncheck this box if you do not want to export all addins. If unchecked no addins at all will be exported...</p>
+                </label>
+            </div>
+            <div id="exportedUsers">
+                <h2>Users</h2>
+                <div class="description">You have <span class="bold">not exported any users</span>.</div>
+            </div>
+            <div id="exportedZones">
+                <h2>Zones</h2>
+                <div class="description">You have <span class="bold">not exported any zones</span>.</div>
+                <label id="exportAllZones">
+                    <input id="export_all_zones_checkbox" value="export_all_zones" type="checkbox" class="checkboxStyle" checked></input>
+                    <span class="infoText">Export all zones</span>
+                    <p><b>Note:</b> Uncheck this box if you do not want to export all zones. If unchecked only zones linked (with dependencies) will be exported...</p>
+                </label>
+            </div>
+            <div id="exportSystemSettings">
+                <h2>System Settings</h2>
+                <div class="description">You have <span class="bold">not exported any system settings.</span>.</div>
+                <label id="exportSystemSettings">
+                    <input id="export_system_settings_checkbox" value="export_system_settings" type="checkbox" class="checkboxStyle" checked></input>
```

Comment:
remove "</input>" at the end of lines #52, #65, #74.

Reply:
I dont understand?

Reply:
done